src/component/SpecificArticle.jsx

```diff
@@ -0,0 +1,30 @@
+import React, { useState, useEffect } from "react";
+import { useParams } from "react-router-dom";
+import Articles from "../modules/articles";
+import { Container } from "semantic-ui-react";
+
+const SpecificArticle = () => {
+  const [article, setArticle] = useState({});
+  const { id } = useParams();
+
+  useEffect(() => {
+    const getSingleArticle = async () => {
+      const response = await Articles.show(id);
+      setArticle(response);
+    };
+
+    getSingleArticle();
+  }, [id]);
+
+  return (
+    <Container textAlign="center">
+    <div data-cy="article">
```

Comment:
please fix the indentation here

Reply:
Fixed.

Reply:
> very minimal styling here, you need to get this styling approved from the rest of the team before I can merge it

We were all in a meet and decided it would be best to focus on the more complex things while we have support and we can focus on styling and the easier things over the weekend.

Reply:
@grconnor We said in the beginning of this project that we expect you to style with each feature and every feature should be ready to be delivered to 'real users'. It doesn't have to be the final touch, but just some styling that corresponds to your Lo-Fi. Otherwise you have to do a PR for only styling and we want to avoid that as much as possible. So please look at your Lo-fi

Reply:
@emtalen We have added some styling to the specific article but we were wanting to keep the content part clean and style more on the Navbar and that when we implement them.

![Screen Shot 2020-10-09 at 14 49 59](https://user-images.githubusercontent.com/48648789/95585107-0d763480-0a3f-11eb-9d83-4e1857778654.png)